NEWS.md

```diff
@@ -3,6 +3,7 @@
 * Removed UUID string pattern from RAML file, as module now uses externalSystemID to look up patron data [EDGPATRON-61] (https://issues.folio.org/browse/EDGPATRON-61)
 * Include API key parameter for every endpoint in RAML. [EDGPATRON-63](https://issues.folio.org/browse/EDGPATRON-63)
 * Update Log4j to 2.16.0. (CVE-2021-44228) [EDGPATRON-67](https://issues.folio.org/browse/EDGPATRON-67)
+* Update Log4j to 2.17.0. (CVE-2021-45105) [EDGPATRON-74](https://issues.folio.org/browse/EDGPATRON-74)
```

Comment:
I don't think it is necessary to have two separate lines of releases notes for these